samples/wifi/wfa_qt_app/src/qt_app_main.c

```diff
@@ -133,6 +136,10 @@ int main(void)
 
 	net_config_init_app(dev, "Initializing network");
 #endif
+	ret = wpa_supp_events_init();
```

Comment:
Please divide in to two separate APIs: It will be clear what the API is doing? IMHO, I don't think you need to wait here blocking the main thread, block the QT thread until WPA supplicant is ready.

```suggestion
	ret = wpa_supp_events_register();
	ret = wait_for_wpa_s_ready();
```